---
name: Gabriel-Mateus
surname: Bernardo Harrington
position: "Research Associate"
address: "School of Medicine, Dementia Research Institue"
#phone: +44 077 390 47519
profilepic: avatar.jpeg
www: gmbernardoharrington.netlify.app
email: "Bernardo-HarringtonG@cardiff.ac.uk"
#twitter: Hmateus4
github: H-Mateus
linkedin: gmb-harrington
orcid: 0000-0001-6075-3619
date: "`r format(Sys.time(), '%B %Y')`"
aboutme: "Currently a Research Associate as a bioinformatician at the [Dementia Research Institute](https://ukdri.ac.uk/), [Cardiff University](https://www.cardiff.ac.uk/), working on Alzheimer's disease with a focus towards genomics."
#output: vitae::twentyseconds
output: vitae::awesomecv
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, warning = FALSE, message = FALSE)

## load packages, xfun will install packages if they aren't installed already
require(xfun)

packages <- c("vitae",
              "tibble",
              "rorcid",
              "readr",
              "dplyr",
              "lubridate",
              "glue",
              "here"
)

xfun::pkg_attach2(packages, message = FALSE)

## read in data
df <- read_csv(here("cv_basis/awarded_funding.csv")) %>%
  dplyr::mutate(start_year = lubridate::year(start), 
                end_year = lubridate::year(end))
```

```{r}
skills <- tribble(
    ~area, ~skills,
    "Programming Languages", "R, BASH, Python, SQL",
    "Bioinformatics", "Proteomics, Genomics, High performance cluster computing, Electronic Health data",
    "Wet Lab work", "3D Tissue Culture, Microscopy, Animal handling, Histology",
    "Markup Languages", "Markdown, RMarkdown/Quarto, YAML, CSS, HTML, LaTeX",
    "Version Control", "Git, GitHub, GitLab",
    "Language", "Portuguese",
    "Microsoft Office", "Excel, Outlook, OneNote, PowerBI, PowerPoint, Visio, Word"
)
```

```{r load-orcid-data}
## education data
orcid_education_data <- do.call("rbind",
  rorcid::orcid_educations("0000-0001-6075-3619")$`0000-0001-6075-3619`$`affiliation-group`$summaries
)
## awards data
orcid_distinctions_data <- do.call("rbind",
  rorcid::orcid_distinctions("0000-0001-6075-3619")$`0000-0001-6075-3619`$`affiliation-group`$summaries
)
## funding data
orcid_funding_data <- do.call("rbind",
  rorcid::orcid_fundings("0000-0001-6075-3619")$`0000-0001-6075-3619`$group$`funding-summary`
)
```

# Research profile

In my current role I proposed, built and maintain a PostgreSQL database of multiple large cohorts spanning over 20 years of data, which has enabled much more efficient and accurate access to this data, facilitating new and ongoing research projects.
I have also built several extensible, well-documented pipelines for processing and analysis of data from multiple instruments and assays in a transparent, robust and reliable fashion.
Most recently, I have proposed and lead on a push for more reproducible practises in research with colleagues, including the use of version control, project management strategies, containerisation and more. 
As part of this push I have also been promoting the use of registered reports to counter [P-Hacking and HARKing](https://simplifaster.com/articles/p-hacking-harking-scientific-replication/).

My PhD at Keele University was based in the [OsKOR](https://oskor.netlify.app/) group at The [RJAH Orthopaedic Hospital](https://www.rjah.nhs.uk/) and focused on spinal cord injury (SCI).
I generated statistical models of SCI using [routine haematological data from electronic health records](https://doi.org/10.1089/neu.2020.7144) and examined the [plasma proteome of human SCI patients](https://doi.org/10.1101/2022.07.12.499696) to investigate biomarkers of neurological outcomes which is current unpredictable in all but the most severely injured patients.
Lab rotations in my first year allowed me develop a range of bench skills, and exposed me to a diverse range of teams and working environments.

# Education

```{r}
orcid_education_data %>%
  detailed_entries(
    what = `education-summary.role-title`,
    when = glue::glue("{`education-summary.start-date.year.value`} - {`education-summary.end-date.year.value`}"),
    with = `education-summary.organization.name`,
    where = `education-summary.organization.address.city`
  )
```

# Awards

```{r}
#tribble(
#  ~Year, ~Type, ~Desc,
#  1903, "Physics", "Awarded for her work on radioactivity with Pierre Curie and Henri Becquerel",
#  1911, "Chemistry", "Awarded for the discovery of radium and polonium"
#) %>% 
#  brief_entries(
#    glue::glue("Nobel Prize in {Type}"),
#    Year, 
#    Desc
#  )
```

```{r}
orcid_distinctions_data %>%
  detailed_entries(
    what = glue::glue("{`distinction-summary.department-name`} - {`distinction-summary.role-title`}"),
    when = `distinction-summary.start-date.year.value`,
    with = `distinction-summary.organization.name`,
    where = `distinction-summary.organization.address.city`
  )
```

# Funding

```{r}
df %>% 
  dplyr::filter(type == "funding") %>%
  detailed_entries(
    what = what,
    when = glue::glue("{start_year}"),
    with = with,
    where = city,
    why = glue::glue("Â£{amount} awarded")
  )
```

# Talks

```{r}
df %>%
  dplyr::filter(type == "talk") %>%
  detailed_entries(
    what = what,
    when = start_year,
    with = with,
    where = city
  ) %>%
  dplyr::arrange(desc(when))
```

# Skills

```{r skills}
skills %>%
  detailed_entries(
    with = area,
    what = skills
  )
```

# Publications

```{r}
library(dplyr)
#knitr::write_bib(c("vitae", "tibble"), "packages.bib")

bibliography_entries(here("cv_basis/publications.bib")) %>%
  arrange(author$family, issued)
```
